Jellyfin.Plugin.Webhook/Helpers/DataObjectHelpers.cs

```diff
@@ -103,6 +103,7 @@ public static class DataObjectHelpers
                 if (episode.IndexNumber is not null)
                 {
                     dataObject["EpisodeNumber"] = episode.IndexNumber;
+                    dataObject["SeasonId"] = episode.SeasonId;
```

Comment:
I would add this to a separate null check since it isn't directly related to the IndexNumber

Reply:
I tried that but then I get this error:

```Cannot convert null to 'Guid' because it is a non-nullable value type```

Reply:
'''
if(!episode.SeasonId.Equals(default))
'''

Reply:
This should be better then